source/includes/language-compatibility-table-csharp.rst

```diff
@@ -10,6 +10,13 @@
      - .NET 4.8
      - .NET 4.7 [#2.14-note]_
      - .NET 4.6
+   * - 2.25
```

Comment:
Suggestion: Condense rows 2.20 through 2.25 to a single row, similar to Go.

Reply:
Looks like 2.14 through 2.25 are the same, so I'll do it for those.